nf_core/pipeline-template/{{cookiecutter.name_noslash}}/main.nf

```diff
@@ -23,7 +23,7 @@ def helpMessage() {
     Mandatory arguments:
       --reads [file]                Path to input data (must be surrounded with quotes)
       -profile [str]                Configuration profile to use. Can use multiple (comma separated)
-                                    Available: conda, docker, singularity, test, awsbatch and more
+                                    Available: conda, docker, singularity, test, awsbatch, <institute> and more
```

Comment:
```suggestion
                                    Available: conda, docker, singularity, test, <institute> and more
```
Isn't `awsbatch` an institute profile now?

Reply:
It's loaded through nf-core/configs, but it's not really an institute still. We've removed it in some places because it's not bundled with the pipeline, but I'm ok with leaving it here as it should basically always be available to the end user and it's not an institute as such...